server/views/pages/index.ejs

```diff
@@ -38,7 +38,7 @@
                       Establishment
                     </li>
                     <li>
-                      Personal mobile number, or personal email address (or both) – we will then use these to send you notifications
+                      Personal email address – we will then use this to send you notifications
```

Comment:
Should have been changed along with the banner about the removal of SMS for notifications